website/docs/cloud-docs/workspaces/organize-workspaces-with-projects.mdx

```diff
@@ -0,0 +1,95 @@
+---
+page_title: Organizing Workspaces with Projects
+description: >-
+  Use projects to organize and group workspaces and create ownership boundaries across your infrastructure.
```

Comment:
Could we use another word instead of "boundaries" so we don't create search collisions with Boundary?